helix-term/src/commands.rs

```diff
@@ -1758,6 +1759,7 @@ fn goto_mode(cx: &mut Context) {
             match (doc.mode, ch) {
                 (_, 'g') => move_file_start(cx),
                 (_, 'e') => move_file_end(cx),
+                (_, 'm') => match_brackets(cx),
```

Comment:
`g%` seems too hard to type, how about `mm` or `mb` for "matching bracket" ?

Reply:
I think `mm` works for me because it's easy to just double tap. `mb` is indeed a good fit for "match brace" e.g. any closest matching brace (`(`/`[`, etc)

Reply:
I'll add `mm` then. As an aside, it'd be better if it could jump to the brace from anywhere inside of two braces as opposed to when only being _on_ the braces themselves.

Reply:
I was considering that as well but followed vim behavior first so it was predictable where the cursor jumped. I'd be open to change it though!